_test/bundle_test.rb

```diff
@@ -50,20 +56,29 @@ def create_jekyll_script
       ].join("\n"))
     end
 
+    def exec_go_script(arg, **options)
+      if ENV['COMSPEC']
```

Comment:
It's the command shell on Windows. This is a way of being able to tell if the `go_script` can be run directly, or must be passed as the first argument to the ruby interpreter.